homeTask_5/homeTask_5/Book.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace homeTask_5
+{
+    class Book
+    {
+        public String Name { get; set; } = "";
+        public int PageCount { get; set; }
+        public String UniqueId { get; set; } = "";
+        public Book(String name, int pageCount, String uniqueId)
+        {
+            this.Name = name;
+            this.PageCount = pageCount;
+            this.UniqueId = uniqueId;
+        }
+        public Book()
+        {
+
+        }
+       
+        public void GetBook()
```

Comment:
Incorrect method naming. I think we print info instead of get book object.

Reply:
> Incorrect method naming. I think we print info instead of get book object.
Done!